unlock-app/src/reducers/walletStatusReducer.js

```diff
@@ -19,6 +23,10 @@ const walletStatusReducer = (walletStatus = initialState, action) => {
     return initialState
   }
 
+  if (action.type === DISMISS_CHECK) {
```

Comment:
could you move the `DISMISS_CHECK` and `GOT_WALLET` into the catch-all at line 14?